### PR TITLE
UCHV-62 - added the complainants step

### PR DESCRIPTION
**Pull request by gxxm**
* added the step field and section to the route config
* added addition skeleton steps for the personal and contact detail to the route config
* added rule to eslint in acceptance tests for generator function
* add field mixins and validation to field config
* added the field and validation translatioins
* added the acceptance tests for the step

### CODE REVIEW

**Comment by joefitter** [comment]
@gxxm Jira isn't picking this up as the commit has the wrong ticket number. Its UCHV (for some bizarre reason). Can we amend the commit message so it gets picked up?

File: acceptance_tests/features/applicant-name.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    applicantNamePage.id['applicant-family-name']
+  ]);
+});
+
+Scenario('On submitting the completed step I am taken to applicants DOB step', (
+  I,
+  applicantNamePage,
+  applicantDOBPage
+) => {
+  I.seeElement(applicantNamePage.id['applicant-given-name']);
```
Comment (joefitter) [inline acceptance_tests/features/applicant-name.js]: Do we need the seeElement? It is covered by the filling method below, if it isn't there we cant fill it

Comment (joefitter) [inline acceptance_tests/features/applicant-name.js]: Also there is a separate test further up that checks correct elements are present

File: acceptance_tests/features/applicant-name.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  I.fillField(applicantNamePage.id['applicant-given-name'], applicantNamePage.content['applicant-given-name']);
+  I.seeElement(applicantNamePage.id['applicant-family-name']);
```
Comment (joefitter) [inline acceptance_tests/features/applicant-name.js]: as above

**Comment by joefitter** [comment]
Thats a 👍  from me

**Comment by joefitter** [comment]
@bendiggle @easternbloc @JoeChapman

**Comment by bendiggle** [comment]
LGTM

**Pull request merged**